src/components/waste/WasteList.tsx

```diff
@@ -39,6 +39,7 @@ export const WasteList = ({ data, ListHeaderComponent, query, selectedTypes }: W
 const styles = StyleSheet.create({
   feedbackContainer: {
     justifyContent: 'flex-end',
-    marginTop: normalize(10)
+    marginTop: normalize(10),
+    marginBottom: normalize(70)
```

Comment:
if there is not export button, there should not be the bottom margin

<img src="https://github.com/user-attachments/assets/15db6ceb-c950-40f7-8c84-2def36fff823" width="200" />